docs/styles.less

```diff
@@ -1,6 +1,7 @@
 @import '~bootstrap/less/bootstrap.less';
 @import '~prismjs/themes/prism.css';
 @import '../src/styles/ux-aspects.less';
+@import '../src/styles/hpe-icons.css';
```

Comment:
In the plunker you are adding this before ux-aspects.css. Whichever way around they are imported, it should be consistent.

Reply:
Yes this works ok both ways.